### ISSUE TITLE
Bug icons color with links <a>

### ISSUE THREAD

**Issue by ice-blaze** [oblique-bit/oblique#73]
Hello Oblique team,

With oblique 8, icons inside a `<a>` seems to always keep the color black.

```html
<a mat-icon-button obButton="primary" color="primary">
	<mat-icon svgIcon="calendar"></mat-icon>
</a>
<br />
<button mat-icon-button obButton="primary" color="primary">
	<mat-icon svgIcon="calendar"></mat-icon>
</button>
```
![image](https://user-images.githubusercontent.com/3356843/181005284-2f9ec591-b831-4ade-964c-b4c2390feeb9.png)

The first one stays black the second one is blue. 

By the way in the documentation it's written `Furthermore, the ObButton automatically sets the color input to primary, meaning that it should not be manually set.` but in my case I have always to specify the color otherwise it's black.

**Comment by c-saner** [comment]
Hello,

Thank you for your report.

We'll look into this issue. Here is the ticket number on our internal system: OUI-1838.

Kind regards

**Comment by c-saner** [comment]
Hello,

This issue was fixed with Oblique 9.0.1. 

It's due to the absence of href or routerLink attribute.

Kind regards
